avm/res/event-grid/topic/tests/e2e/waf-aligned/main.test.bicep

```
@@ -0,0 +1,143 @@
+targetScope = 'subscription'
+
+metadata name = 'Using large parameter set'
+metadata description = 'This instance deploys the module with most of its features enabled.'
```

```suggestion
metadata name = 'WAF-aligned'
metadata description = 'This instance deploys the module in alignment with the best-practices of the Azure Well-Architected Framework.'
```

Note: This requires an update of the readme